governance/polls/Proposal - Rates V2 Poll - October 12, 2020.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Rates V2 Poll - October 12, 2020
+summary: Signal your support to execute the Rates V2 plan.
+discussion_link: https://forum.makerdao.com/t/signal-request-rates-v2/4481
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Rates V2 Poll - October 12, 2020
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) which gives Maker Governance the chance to signal their support for the implementation of Rates V2 plan.
+
+This non-standard weekly Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-mcd-faqs/faqs#governance)) will be active for 3 days beginning on Monday, October 12 at 16:00 UTC.
+
+## Review
```

Comment:
I think we should just refer people to the signal thread rather than having the whole proposal here. Maybe cut it down to just the 'plan' part? I did something similar with the collateral onboarding poll. Cutting out the reasoning and just including the actual proposed changes.